### DISCUSSION TITLE
How can outlines/contours be displayed using <model-viewer>?

### DISCUSSION THREAD

**Discussion by k-p-1** [google/model-viewer#4693] [Q&A]
Hi, 
I would like to display the outline lines for my 3D objects in model-viewer, as seen in the photo below. How can I achieve this? 
I work in Rhino3D and export my files via Blender as a glb file. Is there a setting in Blender that I need to adjust before exporting, or a command in model-viewer?
It would be helpful if I could adjust the line thickness and color as well.
Thanks! 
<img width="352" alt="Bildschirmfoto 2024-03-01 um 20 18 10" src="https://github.com/google/model-viewer/assets/161880897/7fcb9993-1309-4c84-a90e-192cea30385e">
<img width="371" alt="Bildschirmfoto 2024-03-01 um 20 18 34" src="https://github.com/google/model-viewer/assets/161880897/1b5fc32b-e26e-43e4-99bd-1c2883c2cd44">

**Comment by elalish** [comment]
We don't support that, as we only do Physically-Based Rendering. You might be able to do this in three.js, in which case you can fork us to add it.

**Comment by george-martinec** [comment]
Please add
